package.json

```
@@ -18,16 +18,16 @@
     ]
   },
   "dependencies": {
-    "@hapi/hoek": "9.x.x",
-    "@hapi/teamwork": "5.x.x",
-    "@hapi/validate": "1.x.x"
+    "@hapi/hoek": "^10.0.0",
+    "@hapi/teamwork": "^6.0.0",
+    "@hapi/validate": "^2.0.0"
   },
   "devDependencies": {
-    "@hapi/code": "8.x.x",
+    "@hapi/code": "^9.0.0",
     "@hapi/eslint-plugin": "*",
-    "@hapi/lab": "24.x.x",
+    "@hapi/lab": "^25.0.0",
```

Pending https://github.com/hapijs/lab/pull/1051
```suggestion
    "@hapi/lab": "^25.0.1",
```